### DISCUSSION TITLE
We should make it available for Mac users

### DISCUSSION THREAD

**Discussion by fdfdfdfdg** [ScratchAddons/ScratchAddons#5702] [Ideas]
I have a Mac, and I desperately need Addons. I deleted code I don't want deleted and I can't remember what code it is.

**Comment by BroJac5246** [comment]
The issue is that there's no way we're paying Apple $99 a year for a developer account. We don't have that kind of money and wouldn't get the extension to enough users to justify the price, and it wouldn't even work on Safari without modifications. Instead of supporting Apple and its anti-competitive practices, if I were you, then I would just switch to Chrome or Firefox, which are better and more developer-friendly.

**Comment by Samq64** [comment]
Related: #3452

**Comment by DNin01** [comment]
Update - someone on the team offered to publish Scratch Addons to the App Store for us. See https://github.com/ScratchAddons/ScratchAddons/discussions/3452#discussioncomment-7941734.
